src/components/form/TextInputElement/TextInputElement.tsx

```diff
@@ -84,21 +85,30 @@ type InitialState = Pick<
     | 'query'
 >;
 
-const KEY_AT = 50;
-const KEY_SPACE = 32;
-const KEY_ENTER = 13;
-const KEY_UP = 38;
-const KEY_DOWN = 40;
-const KEY_TAB = 9;
-const KEY_P = 80;
-const KEY_N = 78;
-const KEY_ESC = 27;
-const KEY_BACKSPACE = 8;
-
-export const MAX_GSM_SINGLE = 160;
-export const MAX_GSM_MULTI = 153;
-export const MAX_UNICODE_SINGLE = 70;
-export const MAX_UNICODE_MULTI = 67;
+const KEY_AT: number = 50;
+const KEY_SPACE: number = 32;
+const KEY_ENTER: number = 13;
+const KEY_UP: number = 38;
+const KEY_DOWN: number = 40;
+const KEY_TAB: number = 9;
+const KEY_P: number = 80;
+const KEY_N: number = 78;
+const KEY_ESC: number = 27;
+const KEY_BACKSPACE: number = 8;
+
+export const MAX_GSM_SINGLE: number = 160;
+export const MAX_GSM_MULTI: number = 153;
+export const MAX_UNICODE_SINGLE: number = 70;
+export const MAX_UNICODE_MULTI: number = 67;
```

Comment:
Isn't number implicit here?

Reply:
doh, yeah, I got into the habit of annotating everything at some point. happy to remove